src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -42,19 +43,22 @@ def get_validated_dmatrices(train_path, validate_path, content_type, csv_weights
     :param csv_weights: 1 if instance weights are in the second column of csv data files; otherwise, 0
```

Comment:
missing doc string for ```is_pipe```

Reply:
Resolved, also added doc strings for the new data utils.